Themes/Inspired by Things (Dark).theme.json

```diff
@@ -0,0 +1,121 @@
+{
+    "name": "Inspired by Things",
+    "style": "Dark",
+    "author": [
+        {
+            "name": "Matt Stratford",
+            "twitter": "@mattstratford"
+        }
+    ],
+    "version": "0.1",
+
+    "editor": {
+        "backgroundColor": "#1D2229",
+        "caretColor": "#FE3A6F",
+        "selection": {
+            "backgroundColor": "#6BAEF8",
+            "unfocusedBackgroundColor": "#CAE2FE"
+        },
+        "highlight": {
+            "backgroundColor": "#5694F9",
+            "unfocusedBackgroundColor": "#F04E98"
+        }
+    },
+
+    "savedSearchesSidebar": {
+        "color": "#FFFFFF",
+        "backgroundColor": "#1D2229",
+        "highlightedBackgroundColor": "#FED530",
+        "border": false,
+        "iconColors": {
+            "default": "#6BAEF8"
+        }
+    },
+
+    "resultsList": {
+        "color": "#FAF4D1",
+        "backgroundColor": "#1D2229",
+        "alternateBackgroundColor": "#1D2229",
+        "selection": {
+            "color": "#383B3F",
+            "backgroundColor": "#CAE2FE",
+            "unfocusedColor": "#383B3F",
+            "unfocusedBackgroundColor": "#E7EAED"
+        },
+        "border": "#6BAEF8",
+    },
+
+    "styles": {
+        "base": {
+            "color": "#FFFFFF",
+            "font-family":"iA Writer Quattro V, Courier, sans-serif"
```

Comment:
Note that `font-family` will be ignored because The Archive doesn't recognize that key -- but to be extra safe, please remove lines like this here, below, and in the other file.

You can set the `"font"`, but that's not 'cascading', i.e. you have to pick 1 font instead of a list of fallbacks.